apps/server/src/modules/provisioning/dto/provisioning-user-output.dto.ts

```diff
@@ -11,11 +11,12 @@ export class ProvisioningUserOutputDto {
 		this.lastName = user.lastName;
 		this.roleNames = user.roleNames;
 		this.schoolId = user.schoolId;
+		this.externalId = user.externalId;
 	}
 
 	id?: EntityId;
 
-	email: string;
+	email?: string;
```

Comment:
Why is it optionally?

Reply:
Becauase the SANIS user info endpoint does not provide an email.

Reply:
For OAuth2 Users we don't need an email address. So it is optional here